go.mod

```diff
@@ -0,0 +1,85 @@
+module github.com/containerd/cri
+
+go 1.13
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
+	github.com/Microsoft/hcsshim v0.8.7-0.20191021170233-d2849cbdb9df
+	github.com/cilium/ebpf v0.0.0-20191203103619-60c3aa43f488 // indirect
+	github.com/containerd/cgroups v0.0.0-20191206154412-fada802a7909
+	github.com/containerd/console v0.0.0-20191219165238-8375c3424e4d // indirect
+	github.com/containerd/containerd v1.3.1-0.20200109164906-0a1f2b40642e
+	github.com/containerd/continuity v0.0.0-20190815185530-f2a389ac0a02
+	github.com/containerd/fifo v0.0.0-20190816180239-bda0ff6ed73c
+	github.com/containerd/go-cni v0.0.0-20190822145629-0d360c50b10b
+	github.com/containerd/go-runc v0.0.0-20191206163734-a5c2862aed5e // indirect
+	github.com/containerd/ttrpc v0.0.0-20190828172938-92c8520ef9f8 // indirect
+	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20
+	github.com/containernetworking/plugins v0.7.6
+	github.com/davecgh/go-spew v1.1.1
+	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/docker v1.4.2-0.20191121165722-d1d5f6476656
+	github.com/docker/go-events v0.0.0-20170721190031-9461782956ad // indirect
+	github.com/docker/go-metrics v0.0.0-20180131145841-4ea375f7759c // indirect
+	github.com/gogo/googleapis v1.2.0 // indirect
+	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
+	github.com/golang/protobuf v1.3.1
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/imdario/mergo v0.3.7 // indirect
+	github.com/json-iterator/go v1.1.8 // indirect
+	github.com/opencontainers/go-digest v1.0.0-rc1.0.20180430190053-c9281466c8b2
+	github.com/opencontainers/image-spec v1.0.1
+	github.com/opencontainers/runc v1.0.0-rc9
+	github.com/opencontainers/runtime-spec v1.0.2-0.20190207185410-29686dbc5559
+	github.com/opencontainers/selinux v1.2.2
+	github.com/pkg/errors v0.8.1
+	github.com/sirupsen/logrus v1.4.2
+	github.com/stretchr/testify v1.4.0
+	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 // indirect
+	github.com/tchap/go-patricia v2.2.6+incompatible // indirect
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
+	golang.org/x/sys v0.0.0-20191120155948-bd437916bb0e
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	google.golang.org/grpc v1.23.1
+	k8s.io/apimachinery v0.16.5-beta.1
+	k8s.io/client-go v0.0.0
```

Comment:
v0.0.0 is very unexpected here. I would have expected matching versions of client-go, api, apimachinery, and cri-api